Date/MomentFormatConverter.php

```diff
@@ -37,6 +37,31 @@ class MomentFormatConverter
         "d.m.Y, H:i:s"               => 'DD.MM.YYYY, HH:mm:ss',  // german format: 14.05.2014, 13:55:01
         "yyyy-MM-dd"                 => 'YYYY-MM-DD',            // 2014-05-14
         "Y-m-d"                      => 'YYYY-MM-DD',            // 2014-05-14
+
+        // 24 hour format
+        'G:i'                        => 'H:mm',
```

Comment:
Seems that your are converting time from php `date` format. The [Intl formats](http://userguide.icu-project.org/formatparse/datetime#TOC-Date-Time-Format-Syntax) should be used instead.